### ISSUE TITLE
How to install/run Synthpop

### ISSUE THREAD

**Issue by shanwenainimen** [UDST/synthpop#27]
Hi all,

Could anyone please provide some guidance about how to install or run Synthpop? I have been recently assigned a task related with Synthpop and had a hard time doing this job. I would really appreciate it.

**Comment by jiffyclub** [comment]
How familiar are you with using Python and Python libraries?

**Comment by shanwenainimen** [comment]
I took a Python course during undergraduate and have the basic knowledge for Python and its libraries

**Comment by jiffyclub** [comment]
Okay, great. The only way to use SynthPop is by writing Python. :smile:

Step one is to install Python and all the dependencies. I recommend Anaconda. The required dependencies are:

- [census](https://pypi.python.org/pypi/census)
- numexpr
- numpy
- pandas
- scipy
- [us](https://pypi.python.org/pypi/us)

Everything except cenus and us are included with Anaconda.

Step 2 is to download the source code from here on GitHub. Install SynthPop by running `python setup.py install` in the synthpop directory.

Then you should be able to start Python and import synthpop. For usage help I recommend looking at the [demos](https://github.com/UDST/synthpop/tree/master/demos).

**Comment by shanwenainimen** [comment]
Thanks a lot, I have installed Anaconda and I will follow your guidance. Thanks again!

**Comment by waddell** [comment]
If you need help with the logic behind it or possibly some assistance in applying it, you might try reaching out to Karthik Konduri who wrote the precursor to this version, called PopGen.

**Comment by waddell** [comment]
Or https://github.com/bhargavasana..

**Comment by shanwenainimen** [comment]
Thanks @jiffyclub and @waddell, I have successfully installed Census and US, but how can I install/run the synthpop via Jupyter Notebook. I try to run set up.py but it shows error below:
![image](https://cloud.githubusercontent.com/assets/15131880/10954547/1ad225b2-831c-11e5-86fc-d47f00064d01.png)
Thank you so much for your kind input.

**Comment by jiffyclub** [comment]
You'll need to install synthpop from the command line after downloading the source.

**Comment by shanwenainimen** [comment]
I really appreciate your help @jiffyclub  and It works. when I run the census_api.py in deom,  it shows error below:
![image](https://cloud.githubusercontent.com/assets/15131880/10982539/d6939118-83db-11e5-941f-31fbbfba9195.png)
I got the API key and should I just replace the os.environ("census") with key as shown below:
![image](https://cloud.githubusercontent.com/assets/15131880/10982602/274a5fb0-83dc-11e5-8f0a-d3c906e11f16.png)
Thanks

**Comment by fscottfoti** [comment]
Yes that works.  It seems like you've already done that - is it working?

**Comment by shanwenainimen** [comment]
Yes, it works. Many thanks

**Comment by shanwenainimen** [comment]
Hi all,

I have successfully run examples in Demo file thanks to your generous help. I have another question. How to test input_data in Demo file (I assume this is an example used in Popgen)? Because I have to finally set up Synthpop and UrbanSim for our MPO, should I make the format of our data exactly same as data in input_data file? Many thanks.
